src/main/java/ChatReques5.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
```

Comment:
Redundant annotation. The @Data annotation already includes @Getter, @Setter, @ToString, and @AllArgsConstructor. Consider removing redundant annotations to reduce clutter.